mirdata/datasets/mtg_jamendo_autotagging_moodtheme.py

```diff
@@ -0,0 +1,340 @@
+                "data",
+                "splits",
+                "split-" + str(ii),
+                "autotagging_moodtheme-validation.tsv",
+            )
+            with open(path_validation, "r") as fhandle:
+                d = list(
+                    csv.DictReader(
+                        fhandle,
+                        delimiter="\t",
+                        fieldnames=[
+                            "TRACK_ID",
+                            "ARTIST_ID",
+                            "ALBUM_ID",
+                            "PATH",
+                            "DURATION",
+                            "TAGS",
+                        ],
+                    )
+                )
+                split["validation"] = [m["TRACK_ID"] for m in d[1:]]
+            path_test = os.path.join(
+                self.data_home,
+                "data",
+                "splits",
+                "split-" + str(ii),
+                "autotagging_moodtheme-test.tsv",
+            )
+            with open(path_test, "r") as fhandle:
+                d = list(
+                    csv.DictReader(
+                        fhandle,
+                        delimiter="\t",
+                        fieldnames=[
+                            "TRACK_ID",
+                            "ARTIST_ID",
+                            "ALBUM_ID",
+                            "PATH",
+                            "DURATION",
+                            "TAGS",
+                        ],
+                    )
+                )
+                split["test"] = [m["TRACK_ID"] for m in d[1:]]
+            splits.append(split)
+        return {"metadata": meta, "splits": splits}
+
+    @core.copy_docs(load_audio)
+    def load_audio(self, *args, **kwargs):
+        return load_audio(*args, **kwargs)
+
+    def split_load_tracks(self, num):
+        """Load a MTG_jamendo_autotagging_moodtheme pre-defined split. There are five different train/validation/tests splits.
+        Args:
+             num (int): split to be retrieved from 0 to 4
+        Returns:
+            * dict: {`track_id`: track data} - the train split
+            * dict: {`track_id`: track data} - the validation split
+            * dict: {`track_id`: track data} - the test split
+
+        """
+        if not (0 <= num <= 4):
+            raise Exception("Splits avaiables from num 0 to 4")
+
+        meta = self._metadata
+        train_tracks = {
+            k: v
+            for k, v in self._index["tracks"].items()
+            if k in meta["splits"][num]["train"]
+        }
+
+        validation_tracks = {
+            k: v
+            for k, v in self._index["tracks"].items()
+            if k in meta["splits"][num]["validation"]
+        }
+
+        test_tracks = {
+            k: v
+            for k, v in self._index["tracks"].items()
+            if k in meta["splits"][num]["test"]
+        }
+
+        return train_tracks, validation_tracks, test_tracks
```

Comment:
if the dictionary change gets made above, this function gets much simpler! Also - I think it's better just to give the list of track uris, and not the tracks themselves


```suggestion
        Returns:
            * dict: {"train": [...], "validation": [...], "test": [...]} - the train split

        """
        if not (0 <= num <= 4):
            raise Exception("Splits avaiables from num 0 to 4")

    return self._metadata["splits"][split_number]
```